### DISCUSSION TITLE
QQ群被封啦, 你们赶紧搞个电报群吧

### DISCUSSION THREAD

**Discussion by lxzan** [vertx-china/vertx-web-site#273] [General]
<img width="250" alt="image" src="https://user-images.githubusercontent.com/17956540/154796133-2d236bcd-330d-48f6-822b-42c5ec8a65d4.png">

**Comment by travelerbit** [comment]
赵总建新群了吗

**Comment by lxzan** [comment]
> 赵总建新群了吗

不知道啊

**Comment by okou19900722** [comment]
860910673
